server/templates/pages/src/index.html

```diff
@@ -60,14 +55,7 @@
           {{#t}}Firefox Accounts requires JavaScript.{{/t}}
         </noscript>
         <!--<![endif]-->
-
-        <!-- no build declaration, main.js build
-             is done by requirejs and not concat.
-             rev/usemin will still replace the URL
-             with a cache aware URL.
-         -->
-        <!--[if gt IE 9]><!-->
-        <script data-main="/scripts/main" src="/bower_components/requirejs/require.js"></script>
-        <!--<![endif]-->
+        <script type="text/javascript" src="{{ bundlePath }}/appDependencies.bundle.js"></script>
+        <script type="text/javascript" src="{{ bundlePath }}/app.bundle.js"></script>
```

Comment:
@jrgm what are your thoughts on changing the `rev`ing of assets to this new format: 

![image](https://user-images.githubusercontent.com/128755/35662470-51775704-06e6-11e8-8196-a4289ed6f5e6.png)

## OLD

`https://accounts-static.cdn.mozilla.net/scripts/6e2191f0.main.en.js`

## NEW 

`http://127.0.0.1:3030/bundle-9310570d05bf3e78d74473853c0c53fb3e62e038/app.bundle.en.js`

or

`http://127.0.0.1:3030/scripts/bundle-9310570d05bf3e78d74473853c0c53fb3e62e038/app.bundle.en.js`


Will this cause any problems with the CDN or anything else?

Reply:
From irc

```
[10:42:58]  <vladikoff>	vbudhram, pb, what are your thoughts on this https://github.com/mozilla/fxa-content-server/pull/5868#discussion_r165262520 
[10:45:18]  <pb>	vladikoff: making sure i understand, so everything gets the same hash essentially, which is calculated from the webpack bundle?
[10:45:34]  <vladikoff>	pb, calculated from the git commit sha
[10:45:38]  <vladikoff>	pb, the current commit
[10:45:43]  <pb>	vladikoff: oh, right
[10:45:49]  <pb>	vladikoff: works for me, yep
[10:46:05]  <vladikoff>	pb, shane used dynamic imports to replace require on demand  (https://github.com/tc39/proposal-dynamic-import)
[10:46:55]  <vladikoff>	pb, so the code becomes `import(/* webpackChunkName: "fxaClient" */ 'fxaClient');` and if u cache bust the name then you have to dynamically update the scripts which is super annoying lol
[10:47:25]  <pb>	vladikoff: ah yeah, that would be annoying
[13:24:17]  <jrgm>	As long as the message digest changes when the content (and recursively, all entrained content) changes. 
[13:25:14]  <jrgm>	by version, you mean like v1.111.4/scripts/foo.js? 
[13:26:18]  <vladikoff>	jrgm, yea
[13:27:45]  <jrgm>	While the version form has some style appeal, I kinda prefer using a sha that reflects the actual content as found on the wire. 
```